app/components/SongCard/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ *
+ * MusicCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Image, Typography, Button } from 'antd';
+import { PlayCircleTwoTone, StopTwoTone } from '@ant-design/icons';
+import { colors } from '@app/themes';
+import handleMusic, { actions } from '@app/utils/audioControl.js';
+import { translate } from '../IntlGlobalProvider/index';
+
+const { Paragraph, Title } = Typography;
+
+const Container = styled(Card)`
+  && {
+    width: 25em;
+    height: 32em;
+    background-color: ${colors.songCardBg};
+    display: block;
+    border-radius: 2em;
+    box-shadow: inset -6px -6px 12px rgba(0, 0, 0, 0.8), inset 6px 6px 12px rgba(255, 255, 255, 0.4);
+    overflow-x: scroll;
+    &::-webkit-scrollbar {
+      display: none;
+    }
+  }
+`;
+
+const IconsContainer = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+`;
+
+const CommonIconStyle = {
+  fontSize: 18
+};
+
+const ControlButton = styled(Button)`
+  && {
+    color: white;
+    display: flex;
+    margin: auto 3px;
+    justify-content: space-between;
+    align-items: center;
+    background-color: transparent;
+  }
+`;
+
+export function SongCard({ song }) {
+  const { shortDescription, artworkUrl100, artistName, previewUrl } = song;
+  const playStates = React.useState(false);
+  const play = playStates[0];
+  const songElement = React.useRef(null);
+
+  return (
+    <Container data-testid="song-card">
+      <Image src={artworkUrl100} width="80%" preview="false" height="250px" />
+      <Title style={{ color: '#fafafa', fontSize: 16 }} italic={true}>
```

Comment:
use styled-components please, no stray colors. Please use colors from the colors.js